ci/jenkins/test.sh

```diff
@@ -425,6 +430,170 @@ function deliver_antrea_windows {
     rm -f antrea-windows.tar.gz
 }
 
+function deliver_antrea_windows_containerd {
```

Comment:
Add pre_env function to share env initialization code.
I Also discussed with Wenying about this before, we agreed we can finally have a unified function to implement both containerd and docker building in one function with another PR. But currently there are several different image dependency because we have different version docker and containerd testbeds for windows so we will separate them.